README.md

```diff
@@ -68,19 +68,28 @@ You will also need a Promise polyfill for [older browsers](http://caniuse.com/#f
 We recommend [taylorhakes/promise-polyfill](https://github.com/taylorhakes/promise-polyfill)
 for its small size and Promises/A+ compatibility.
 
-For use with webpack, add this package in the `entry` configuration option
+For use with webpack, either add this package in the `entry` configuration option
 before your application entry point:
 
 ```javascript
 entry: ['whatwg-fetch', ...]
 ```
 
-For Babel and ES2015+, make sure to import the file:
+... or require the file:
+
+```javascript
+require('whatwg-fetch')
+```
+
+For Babel and ES2015+, import the file:
 
 ```javascript
 import 'whatwg-fetch'
 ```
 
+In either case the polyfill sets up a global variable. The module does not export
+anything directly, so do not use the `import fetch from ...` form of the import statement.
```

Comment:
Is this necessary to point out to clarify webpack instructions?

Reply:
I thought it's a good idea to add an explanation as to why both approaches work. Especially after seeing the comments in this closed issue: https://github.com/github/fetch/issues/539

Reply:
Sorry that this never got merged. The library went to some changes since then, and now it exports the `fetch()` method and associated classes. I will update the README documentation to reflect that.